patterns/avd.md

```diff
@@ -113,64 +113,75 @@ Microsoft provide the following profile management solutions:
+* FSLogix – Provides a full profile solution in addition to addressing issues when deploying Office 365 in a non-persistent VDI/RDS environment. Using the Profile Container and/or Office 365 Container data is cached locally helping avoid disruptions or application instability during brief storage interruptions like network switch resets or storage controller failovers. This cache sits between the user’s desktop and the remote container storage (SMB file share or cloud cache functionality) and is configured either to persist between logons or start fresh each time the user logs in.
 * Folder Redirection – Enables certain folders, such as Documents and AppData, to be redirected. This enables user data such as documents and email configuration, to not be loaded at login, which can improve performance when loading profiles. However, some applications communicate with the AppData frequently, making the application appear slow when this folder is redirected.
 
+#### FSLogix considerations
+
+FSLogix provides various functionality and advanced profile configurations that can further optimise the virtual desktop experience:
+
+- Simplification of gold image versioning via the use of application masking. This feature allows the base image to include most optional applications to be installed inside the gold image, while only presenting these applications to authorised users. This simplifies gold image management and application delivery and is relatively simple to setup. For further guidance on this this configuration see [Implement Application Masking Tutorial](https://docs.microsoft.com/en-us/fslogix/implement-application-masking-tutorial).
+
+- Management of Java versioning used for various URLs and applications, for those agencies running multiple java runtimes within the desktop.
+- Use of the **redirections.xml** with profile management provides the ability to control what portions of the profile (in the C: drive) are redirected out in the remote profile and kept in sync. Exclusions can optimise the desktop environment and are sometimes used to make an application work within a virtual environment. Microsoft recommend to use this feature with caution and only include exclusions where the exclusion is fully understood.  This pattern recommends to utilise the crowd sourced **redirections.xml** as a base, and will not maintain specific application recommendations in this pattern. The crowd sourced **redirections.xml** is maintained by the virtual desktop user community and can be found at [crowd sourced Redirections.xml  - github](https://github.com/aaronparker/fslogix/blob/main/Redirections/Redirections.csv).
```

Comment:
Can we also link to MS redirections.xml material/guidance for agencies that wish to use it instead?

What do you mean by the last part? `and will not maintain specific application recommendations in this pattern.`

Reply:
I've updated the guidance to link the minimum recommended from Microsoft and some other thing. 
I've removed the statement around the maintenance of the redirections.xml, but what was meant is that the redirections has been a community thing that is updated based on the broader feedback to the community outside of the blueprint so it doesn't make sense for the blueprint to reference it's own?